zconf-ng.h.in

```diff
@@ -88,6 +88,14 @@
 #  define Z_EXPORTVA
 #endif
 
+/* Conditional exports */
+#ifdef ZLIB_COMPAT
+#  define ZNG_CONDEXPORT Z_INTERNAL
+#else
+#  define ZNG_CONDEXPORT Z_EXPORT
+#endif
```

Comment:
This seems pointless, when using `-ng` variant of headers, `zLIB_COMPAT` is never defined.